COMMITTERS.adoc

```diff
@@ -0,0 +1,173 @@
+= Information for Quarkus Committers
+
+This document contains useful information for Quarkus committers.
+
+[NOTE]
+====
+It is a work in progress, so if you think information is missing/unclear/inadequate,
+please reach out to us on the `quarkus-dev` mailing-list.
+====
+
+This document has a lot of information.
+It is not a set of strict rules, it is here to guide you.
+If you forget to do something, miss something, don't worry, it happens.
+
+Do your best and that will already be a lot :).
+
+== Merging Pull Requests
+
+The first thing is that, while being a committer,
+you can't really commit directly to the main repository:
+you need at least one review by your peers.
+
+Thus the need for a pull request.
+
+So you are more like a merger than a committer
+and you might merge more PRs from others than your own work.
+
+While not absolute, here are some advices:
+
+* if you really want a review by a specific person,
+  make it a named reviewer via a GitHub UI
+* if someone asked for a specific reviewer, better
+  give some time for this person to review the PR.
+  If not reviewed in a reasonable amount of time,
+  better ping them again to have some information
+  or ask the original author if we should go ahead.
+* if in doubt (on a specific part or because you're
+  not comfortable reviewing documentation or any other reason),
+  ask for help
+* except if it's a totally non brainer (typically a typo fix),
+  let the PR bake a few hours so some other people can have a
+  look if interested.
+  If the PR is very large or is a new extension, better let it
+  bake for a few days.
+* we do merge commits so please ask the author to properly
+  squash the commits before merging.
+  The idea is to have proper semantic commits.
+  If the author is not familiar with Git or not available,
+  you can also (carefully) do it for them as soon as you agreed on it.
```

Comment:
Perhaps add this link: https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/committing-changes-to-a-pull-request-branch-created-from-a-fork as a lot of people don't know this is even possible

Reply:
Done.